### PR TITLE
Change axios to use Apollo

### PR DESCRIPTION
**Pull request by virichino**
- Add machine learning UI to work with video converter and machine learning as one
- Add video converter from UI to work with Apollo

### CODE REVIEW

File: ReactUI/src/ui/pages/Converter/ConverterPage.jsx [hunk 1]
```diff
@@ -1,9 +1,25 @@
 import React from "react";
 import SideBar from "../../modules/sidebar/SideBar";
 import Converter from "../../modules/converter/Converter";
+import ApolloClient from "apollo-client";
+import { createUploadLink } from "apollo-upload-client";
+import { ApolloProvider } from "@apollo/react-hooks";
+import { InMemoryCache } from "apollo-cache-inmemory";
 
+const httpLink = createUploadLink({
+	uri: "http://localhost:4000/graphql",
```
Comment (jchoque87) [inline ReactUI/src/ui/pages/Converter/ConverterPage.jsx:10]: You should  work with .env file in this line

File: APOLLO_SERVER2/src/resolvers.js [hunk 2]
```diff
@@ -1,63 +1,196 @@
...
+        machines: async () => {
+            return await FileData;
+        },
+        VideoConverter: async () => {
+            return await FileData1;
+        },
+        files: async () => {
+            const urlCVT = "" + process.env.CONVERTER_GET_DATA;
+            const {data} = await axios.get(urlCVT);
+            console.log({...data}+"here1")
```
Comment (stellith) [inline APOLLO_SERVER2/src/resolvers.js:45]: Delete console log

File: APOLLO_SERVER2/.gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 node_modules
+images
```
Comment (stellith) [inline APOLLO_SERVER2/.gitignore:2]: Newline missing

File: APOLLO_SERVER2/src/typeDefs.js [hunk 4]
```diff
@@ -18,15 +38,34 @@ const typeDefs =  gql`
     Percentage: String
     Second: String
     PathImage: String
+    name: String!
+    filePath: String!
+  }
+
+  type filesMetadata
+  {
```
Comment (stellith) [inline APOLLO_SERVER2/src/typeDefs.js:46]: The keys {} must follow the same format as the previous ones, such as: 
type filesMetadata {
    name: String
    filePath: String
}

File: APOLLO_SERVER2/src/typeDefs.js [hunk 5]
```diff
@@ -18,15 +38,34 @@ const typeDefs =  gql`
...
+    name: String
+    filePath: String
+  }
+
+  type VideoConverter
+  {
```
Comment (stellith) [inline APOLLO_SERVER2/src/typeDefs.js:52]: The keys {} must follow the same format as the previous ones

File: CONVERTER_SERVICE/.gitignore [hunk 6]
```diff
@@ -1 +1,2 @@
-node_modules
+node_modules
+resources
```
Comment (stellith) [inline CONVERTER_SERVICE/.gitignore:2]: Newline missing

File: ReactUI/src/ui/components/materialUI/extractor-service/FormMetadata.jsx [hunk 7]
```diff
@@ -1,92 +1,78 @@
...
+			  file: FileData  
+			}
+		  });
+		  if (error) {
+			console.log(error);
+		  }
+		  else{
+			setResponse(response.data.metaData);
+			setOpen(false);	
+
```
Comment (stellith) [inline ReactUI/src/ui/components/materialUI/extractor-service/FormMetadata.jsx:56]: I should delete this line space

File: ReactUI/src/ui/components/materialUI/extractor-service/FormMetadata.jsx [hunk 8]
```diff
@@ -1,92 +1,78 @@
...
+		  });
+		  if (error) {
+			console.log(error);
+		  }
+		  else{
+			setResponse(response.data.metaData);
+			setOpen(false);	
+			
+		  }
+
```
Comment (stellith) [inline ReactUI/src/ui/components/materialUI/extractor-service/FormMetadata.jsx:58]: I should delete this line space

File: ReactUI/src/ui/pages/Converter/ConverterPage.jsx [hunk 9]
```diff
@@ -1,9 +1,25 @@
...
-	return <SideBar page={Converter} />;
+	return (
+		<ApolloProvider client={client}>
+			<SideBar page={Converter} />
+		</ApolloProvider>
+	);	
 };
 
-export default ConverterPage;
+export default ConverterPage;
```
Comment (stellith) [inline ReactUI/src/ui/pages/Converter/ConverterPage.jsx:25]: Newline missing

**Pull request merged**